src/Symfony/Component/ErrorHandler/DebugClassLoader.php

```diff
@@ -184,7 +184,7 @@ public function getClassLoader(): callable
     public static function enable(): void
     {
         // Ensures we don't hit https://bugs.php.net/42098
-        class_exists(\Symfony\Component\ErrorHandler\ErrorHandler::class);
+        class_exists(ErrorHandler::class);
         class_exists(\Psr\Log\LogLevel::class);
```

Comment:
Should also be imported, right?

Reply:
I applied the [fully_qualified_strict_types](https://cs.symfony.com/doc/rules/import/fully_qualified_strict_types.html) rule, which (with current config) simplifies usage of symbols.

If you want to import symbols, we can enable `import_symbols`.
Please decide if you would like to go this path as separated PR: https://github.com/symfony/symfony/pull/53244